python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -142,3 +150,48 @@ def predict_proba(self, dataset):
         :rtype: numpy.ndarray
         """
         return self._fastai_predict(dataset, 2)
+
+
+class WrappedMlflowAutomlImagesClassificationModel:
```

Comment:
Thanks for adding type checking in common_vision_utils changes. 
I would suggest adding type checking here as well.

Reply:
other methods and classes don't follow it so i didn't add it for consistency.
But i agree that it's better to have it. will add them!